random_message.py

```diff
@@ -10,7 +10,7 @@
            "Sunday 😴"
 ]
 
-random_word = [
+random_words: list[str] = [
     "Hello, it's ",
     "Hey, it's ",
     "Smile, it's ",
```

Comment:
**Suggestion:** Use the typing.List instead of list for type hinting.
```suggestion
random_words: List[str] = [
     "Hello, it's ",
     "Hey, it's ",
     "Smile, it's ",
```